frame/frame.html

```diff
@@ -0,0 +1,7 @@
+<!--
+Copyright 2021 Google LLC
+SPDX-License-Identifier: Apache-2.0
+-->
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>FLEDGE</title>
```

Comment:
Shouldn't the frame run the JavaScript from main?  Or is that for later?

Reply:
The build system actually adds the JavaScript automatically.